app/workers/step_by_step_draft_update_worker.rb

```diff
@@ -1,5 +1,5 @@
 class StepByStepDraftUpdateWorker
-  include Sidekiq::Worker
+  include Sidekiq::Job
```

Comment:
I think we also need to rename and move this file?

> … then rename/move your workers to be app/sidekiq/MyJob.rb instead of app/workers/MyWorker.rb.